po/r-novice-gapminder.ja.po

```diff
@@ -15623,36 +15622,40 @@ msgid ""
 "The commands `jpeg`, `png` etc. are used similarly to produce\n"
 "documents in different formats."
 msgstr ""
-"The commands `jpeg`, `png` etc. are used similarly to produce\n"
-"documents in different formats."
+"同様にして `jepg` や `png` などのコマンドを使うことで、他の形式のドキュメントを作成できます。"
 
 # header
 #: r-novice-gapminder/_episodes/11-writing-data.md:86
 msgid "## Writing data"
-msgstr "## Writing data"
+msgstr "## データの出力"
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:88
 msgid "At some point, you'll also want to write out data from R."
-msgstr "At some point, you'll also want to write out data from R."
+msgstr "R からデータを出力したいこともあります。"
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:90
 msgid ""
 "We can use the `write.table` function for this, which is\n"
 "very similar to `read.table` from before."
 msgstr ""
-"We can use the `write.table` function for this, which is\n"
-"very similar to `read.table` from before."
+"これには、既に使った `read.table` 関数とよく似た `write.table` 関数を使います。"
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:93
 msgid ""
 "Let's create a data-cleaning script, for this analysis, we\n"
 "only want to focus on the gapminder data for Australia:"
 msgstr ""
-"Let's create a data-cleaning script, for this analysis, we\n"
-"only want to focus on the gapminder data for Australia:"
+"データを綺麗にするスクリプトを書きましょう。\n"
```

Comment:
or what about 整形?

Reply:
May be we can leave it as it is. I Googled and "キレイ”, ”きれい” or "クリーン” appear to be used interchangeably. It makes sense, which is most important. Thank you for your attention and great work!

Reply:
Had a quick search to see what's used in Tech industry in Japanese.
[Microsoft Office support](https://support.office.com/ja-jp/article/%E3%83%87%E3%83%BC%E3%82%BF%E3%82%92%E3%82%AF%E3%83%AA%E3%83%BC%E3%83%B3%E3%82%A2%E3%83%83%E3%83%97%E3%81%99%E3%82%8B%E6%96%B9%E6%B3%95%E3%83%88%E3%83%83%E3%83%97-10-2844b620-677c-47a7-ac3e-c2e157d1db19) uses クリーンアップ ("clean up"), although I think クリーン is more concise. It seems that [きれい](https://area.autodesk.jp/column/tutorial/maya_atoz/cleaup-edge/) and [綺麗](http://excel.resocia.jp/report/357/) are also used in the context of data formatting. 

As discussed in our [Guidelines](https://github.com/swcarpentry-ja/i18n/blob/ja/TranslatorGuidelines.md) we prefer to use terms so that learners will be able to understand terminology used elsewhere if they look for help. I think the most important thing is to be consistent, so I recommend to decide on one to use throughout the lessons and add it to the [Glossary](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms) on the Wiki pages. Sorry I know this is a lot of information and our documents are a bit out of date.

In this case I think either is fine, although isn't the Kanji form of 綺麗な rarely used (although I can read it so I'm sure a native speaker can). Stylistically, the きれい could be harder to reader with particles?

```suggestion
"データをキレイにするスクリプトを書きましょう。\n"
```

Of course I think Japanese readers are also familiar with クリーニング:

```suggestion
"データをクレーンにするスクリプトを書きましょう。\n"
```

Thanks for your inputs. I'm OK to leave it in Kanji unless anyone thinks either of these is better.

Reply:
Just curious, is there any reason this wasn't used as it's more literal translation of the English text?
```suggestion
"データクリーニングのスクリプトを書きましょう。\n"
```

Reply:
Since both native speakers seem to agree to leave the Kanji form, I'm going to close this.